morebits.js

```diff
@@ -2984,7 +2983,7 @@ Morebits.wiki.page = function(pageName, currentAction) {
 			query.noredirect = 'true';
 		}
 		if (ctx.watchlistOption === 'watch') {
-			query.watch = 'true';
+			query.watchlist = 'true';
```

Comment:
```suggestion
			query.watchlist = 'watch';
```
nochange, preferences, unwatch, watch are the only allowed values. 

Here and in 4 more uses below.

Reply:
Ugh, braindumb.  Really I think it should be like in the `save`, just `watchlist: ctx.watchlistOption`

Reply:
Now done as in https://github.com/azatoth/twinkle/pull/888#discussion_r392944899.  Only exception is stabilization, which still uses the old system.